docs/directives/README.md

```diff
@@ -116,55 +118,54 @@ Ce n'est pas obligatoire mais aide Vue à mieux comprendre les changements qui s
 <v-for-example-2 />
 
 ::: tip
-Pour répéter un groupe d'éléments, utiliser `v-for` sur une balise `<template>`
+To repeat a group of elements, use `v-for` on a `<template> `tag
 :::
 
-[Exercice: utilisez deux boucles v-for pour afficher tout le contenu du panier](https://codepen.io/sylvainpv/pen/RBqbBW?editors=1010)
+[Exercice: use two v-for loops to display all the contents of the basket](https://codepen.io/sylvainpv/pen/RBqbBW?editors=1010)
 
-## v-on: Événements
+## v-on: Events
 
-Permet de définir une action à effectuer lorsqu'un évènement survient. Il peut s'agir d'un événement du DOM (`click`, `mouseover`, `focus`...) ou d'un événement personnalisé émis par un composant enfant.
+Define an action to take when an event occurs. It can be a DOM event (`click`, `mouseover`, `focus`, etc.) or a custom event emitted by a child component.
 
 ```vue{1,5}
-<button v-on:click="counter += 1"> Cliquez ici ! </button>
-Ce bouton a été cliqué {{ counter }} fois.
+<button v-on:click="counter += 1"> Click here! </button>
+This button has been clicked {{ counter }} times.
 
-<!-- syntaxe raccourcie: @event -->
-<button @click="resetCounter($event)"> Remettre à zéro </button>
+<!-- shortened syntax: @event -->
+<button @click="resetCounter($event)"> Reset </button>
 ```
 
 <v-on-example />
 
 ::: tip
-Vous pouvez utiliser la variable `$event` comme référence à l'événement capturé
+You can use the `$event` variable as a reference to the captured event
 :::
 
-[Exercice: utilisez les événements pour ajouter un singe au clic sur bouton, et leur faire ouvrir les yeux au passage de la souris](https://codepen.io/sylvainpv/pen/NBEKQw?editors=1010)
+[Exercice: use events to add a monkey to the click of a button, and make them open their eyes when the mouse passes](https://codepen.io/sylvainpv/pen/NBEKQw?editors=1010)
```

Comment:
to the click of a button => when clicking the button

when the mouse passes => on mouse hover